test/TaskEither.ts

```diff
@@ -215,31 +214,6 @@ describe('TaskEither', () => {
     })
   })
 
-  describe('getMonoid', () => {
```

Comment:
we can keep the tests related to Semigroup though

Reply:
I added a separate set of `getSemigroup` tests which are equal to the tests from `Either`.